docs/tools/offline/cli-setup.md

```diff
@@ -129,7 +129,8 @@ Mbed CLI requires adding the following to the system `PATH`:
 
 You need to tell Mbed CLI where to find the toolchains that you want to use for compiling. Mbed CLI supports the following toolchains:
 
-- [Arm Compiler 5](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler-5/downloads). Use version 5.06 of Arm Compiler 5. Versions older than 5.06 might be incompatible with the tools.
+- [Arm Compiler 5](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler/downloads/version-5). Use version 5.06 of Arm Compiler 5. Versions older than 5.06 might be incompatible with the tools.
+- [Arm Compiler 6](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler/downloads/version-6). Use version 6.8 of Arm Compiler 6. Versions older than 6.8 might be incompatible with the tools.
```

Comment:
6.10 please.

Reply:
@theotherjimmy Ok, I took that 6.8 number from the Mbed CLI readme, so that will need to get changed too